### PR TITLE
SvelteKit Adapters: Electron and Cordova/Capacitor

### PR DESCRIPTION
**Pull request by ptkdev**

### CODE REVIEW

File: src/routes/components/components.json [hunk 1]
```diff
@@ -2856,5 +2856,25 @@
...
+	},
+	{
+		"title": "SvelteKit Cordova/Capacitor Adapter",
+		"url": "https://github.com/ptkdev/sveltekit-cordova-adapter",
+		"description": "Adapter for build mobile apps with Svelte Kit and Apache Cordova or Ionic Capacitor",
+		"npm": "@ptkdev/sveltekit-cordova-adapter",
+		"addedOn": "2023-02-11T00:00:00Z",
+		"category": "SvelteKit Adapters",
+		"stars": 3,
+		"tags": ["ssg", "mobile apps", "cordova", "capacitor", "ionic", "integrations"]
```
Comment (MacFJA) [inline src/routes/components/components.json]: The tag `ssg` is new.
- It's the adapter that is an SSG or the SvelteKit application ?
- Is it necessary ?
- Can it be added to already existing components ?

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `mobile apps` is new.

- Is it necessary ?
- Can it be added to already existing components ?

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `cordova` is new.

- Is it necessary ?
- Can it be added to already existing components ? (short answer, it seems to be **no**)

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `capacitor` is new.

- Is it necessary ?
- Can it be added to already existing components ? (short answer, it seems to be **no**)

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `ionic` is new.

- Is it necessary ?
- Can it be added to already existing components ? (short answer, it seems to be **no**)

File: src/routes/components/components.json [hunk 2]
```diff
@@ -2856,5 +2856,25 @@
...
+	},
+	{
+		"title": "SvelteKit Electron Adapter",
+		"url": "https://github.com/ptkdev/sveltekit-electron-adapter",
+		"description": "Adapter for build desktop apps with Svelte Kit and Electron",
+		"npm": "@ptkdev/sveltekit-electron-adapter",
+		"addedOn": "2023-02-11T00:00:00Z",
+		"category": "SvelteKit Adapters",
+		"stars": 3,
+		"tags": ["ssg", "desktop apps", "electron", "electron forge", "integrations"]
```
Comment (MacFJA) [inline src/routes/components/components.json]: The tag `desktop apps` is new.

- Is it necessary ?
- Can it be added to already existing components ?

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `electron` is new.

- Is it necessary ?
- Can it be added to already existing components ?

Comment (MacFJA) [inline src/routes/components/components.json]: The tag `electron forge` is new.

- Is it necessary ?
- Can it be added to already existing components ?

File: src/routes/components/components.json (hunk 1 above)
Comment (ptkdev) [reply src/routes/components/components.json]: I use a fork of svelte-kit static-adapter, "ssg" is more accurate.

Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

File: src/routes/components/components.json (hunk 2 above)
Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

Comment (ptkdev) [reply src/routes/components/components.json]: Removed in next commit

File: src/routes/components/components.json (hunk 1 above)
Comment (MacFJA) [reply src/routes/components/components.json]: In that case, the tag should be added to (at least):
- `@sveltejs/adapter-static`
- `@macfja/svelte-adapter-neutralino`
- `sveltekit-adapter-html-like`
- `svelte-adapter-github`
- `svelte-adapter-static-digitalocean`

Comment (ptkdev) [reply src/routes/components/components.json]: Perfect, fixed! Thanks

**Pull request merged**